src/openrct2/actions/LandSetHeightAction.hpp

```diff
@@ -134,11 +134,7 @@ DEFINE_GAME_ACTION(LandSetHeightAction, GAME_COMMAND_SET_LAND_HEIGHT, GameAction
             }
         }
         auto res = std::make_unique<GameActionResult>();
-        res->Cost = 0;
-        if (!(gScreenFlags & SCREEN_FLAGS_EDITOR) && !(gParkFlags & PARK_FLAGS_NO_MONEY))
-        {
-            res->Cost = sceneryRemovalCost + GetSurfaceHeightChangeCost(surfaceElement);
-        }
+        res->Cost = sceneryRemovalCost + GetSurfaceHeightChangeCost(surfaceElement);
```

Comment:
Is this really the only game action that needs modified?

Reply:
Actually FootpathRemoveAction.hpp has GetRefundPrice which doesn't need any of those conditions anymore.